src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets

```diff
@@ -579,15 +580,25 @@
             IntermediateOutputPath="$(_MauiIntermediateImages)"
             InputsFile="$(_ResizetizerInputsFile)"
             Images="@(MauiImage->Distinct())">
+              <Output TaskParameter="CopiedResources" ItemName="_CopiedResources" />
         </ResizetizeImages>
 
         <ItemGroup>
             <!-- Get Images that were generated -->
             <!-- Either from the task, or if the task was skipped (up to date), use the wildcard lookup -->
-            <_ResizetizerCollectedImages Condition="'@(CopiedResources)' != ''" Include="@(CopiedResources)" />
-            <_ResizetizerCollectedImages Condition="'@(CopiedResources)' == ''" Include="$(_MauiIntermediateImages)**\*" />
+            <_ResizetizerCollectedImages Condition="'@(_CopiedResources->Count())' != '0'" Include="@(_CopiedResources)" />
+            <_ResizetizerExistingImages Include="$(_MauiIntermediateImages)\**\*" />
```

Comment:
The only other thing, is it possible for `$(_MauiIntermediateImages)` to *ever* be blank?

If it ever was, this would wildcard your entire drive like `\**\*`.

Reply:
It is set here https://github.com/dotnet/maui/blob/dfba4e83d46ea5e4d7af174ed37520b3e06e1caa/src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets#L71 so it should always be set.

Reply:
Ok, we could rely on it ending with a trailing `\`, then at worst it would wildcard `**\*` in the current directory instead of the whole drive.

But this is kind of unrelated to this change, we don't really have to fix it here.